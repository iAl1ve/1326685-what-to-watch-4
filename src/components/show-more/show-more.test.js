import React from "react";
import renderer from "react-test-renderer";
import ShowMore from "./show-more.jsx";

describe(`Test Render ShowMore button`, () => {
  it(`Render ShowMore button`, () => {
    const tree = renderer
      .create(<ShowMore
        onShowMoreClick = {() => {}}
      />, {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
