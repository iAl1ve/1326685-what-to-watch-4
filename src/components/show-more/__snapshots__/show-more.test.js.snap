// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Render ShowMore button Render ShowMore button 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;
